sbin/prepareWorkspace.sh

```diff
@@ -326,9 +329,9 @@ checkingAndDownloadingAlsa() {
 
 sha256File() {
   if [ -x "$(command -v shasum)" ]; then
-    (shasum -a 256 | cut -f1 -d' ') <$1
+    (shasum -a 256 | cut -f1 -d' ') <"$1"
```

Comment:
Again not an issue for this PR, but I'm curious as to how we ended up with this as opposed to either:
`shasum -a 256 "$1" | cut -f1 -d' '`
or if that cann't work somewhere:
`cat "$1" | shasum -a 256 | cut -f1 -d' '`

Reply:
> shasum -a 256 "$1" | cut -f1 -d' '

I'm not sure either but that seems like a more viable one. Much more readable anyway